app/presenters/tree_builder_containers.rb

```diff
@@ -18,6 +18,6 @@ def set_locals_for_render
 
   # Get root nodes count/array for explorer tree
   def x_get_tree_roots(count_only, _options)
-    count_only_or_objects(count_only, rbac_filtered_objects(Container.all), "name")
+    count_only_or_objects(count_only, rbac_filtered_objects(Container.where(:deleted_on => nil)), "name")
```

Comment:
> Does the RBAC filter not remove the EMS-less items here? Should it?

I just tested it out and looks like it doesn't filter those. Im not sure if it should...